app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -36,8 +36,8 @@ def initializing
 
   def start
     image = target_entity
-    return queue_signal(:abort_job, "no image found", "error") unless image
-    return queue_signal(:abort_job, "cannont analyze non-docker images", "error") unless image.docker_id
+    return queue_abort("no image found") unless image
+    return queue_abort("cannont analyze non-docker images") unless image.docker_id
```

Comment:
Typo: `cannont`

Maybe eliminate the double negative with:  `"can only analyze docker images"`